customer-chat/app.js

```diff
@@ -0,0 +1,196 @@
+/**
+ * Copyright 2017-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ *
+ * Starter Project for Messenger Platform Quick Start Tutorial
+ *
+ * Use this project as the starting point for following the
+ * Messenger Platform quick start tutorial.
+ *
+ * https://developers.facebook.com/docs/messenger-platform/getting-started/quick-start/
+ *
+ */
+
+'use strict';
+
+// Imports dependencies and set up http server
+const
+    request = require('request'),
+    express = require('express'),
+    body_parser = require('body-parser'),
+    app = express().use(body_parser.json()); // creates express http server
+app.use(express.static('public'));
+
+// TODO: Add values here
+const access_token = '';
+const page_inbox_app_id = "";
+
+// Sets server port and logs message on success
+app.listen(process.env.PORT || 5000, () => console.log('webhook is listening'));
+
+// receives webhook events from Messenger Platform
+app.post('/webhook', (req, res) => {
+
+    // parse messaging array
+    const webhook_events = req.body.entry[0];
+
+    // Secondary Receiver is in control - listen on standby channel
+    if (webhook_events.standby) {
+
+        // iterate webhook events from standby channel
+        webhook_events.standby.forEach(event => {
+            handleEvent(event.sender.id, event);
+        });
+    }
+
+    // Bot is in control - listen for messages
+    if (webhook_events.messaging) {
```

Comment:
if standby isn't being used then this if statement probably isn't necessary

Reply:
@amuramoto It is used.